src/main/java/no/fdk/dataservicecatalog/repository/DataServiceMongoRepository.java

```diff
@@ -6,7 +6,7 @@
 import reactor.core.publisher.Mono;
 
 public interface DataServiceMongoRepository extends ReactiveMongoRepository<DataService, String> {
-    Mono<DataService> findByIdAndCatalogId(String dataServiceId, String catalogId);
-    Mono<Long> deleteByIdAndCatalogId(String dataServiceId, String catalogId);
-    Flux<DataService> findAllByCatalogId(String catalogId);
+    Mono<DataService> findByIdAndOrganizationId(String dataServiceId, String catalogId);
+    Mono<Long> deleteByIdAndOrganizationId(String dataServiceId, String catalogId);
+    Flux<DataService> findAllByOrganizationId(String catalogId);
```

Comment:
All occurrences of catalogId can be renamed to organizationId

Reply:
it's called catalogId in url, but organizationId in json body.